python_modules/dagster/dagster/_utils/error.py

```diff
@@ -115,12 +115,11 @@ def serializable_error_info_from_exc_info(
 
     if isinstance(e, DagsterUserCodeExecutionError) and _should_redact_user_code_error():
         error_id = str(uuid.uuid4())
+        masked_logger = logging.getLogger("dagster.masked")
```

Comment:
not having a good first impression with the name `dagster.masked`
riffing 
`dagster.redacted_errors`  
`dagster.private`
`dagster.local_only` 
`dagster.not_upoaded` 
`dagster.hidden`